pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -64,6 +64,7 @@ type Alertmanager interface {
 type MultiOrgAlertmanager struct {
 	Crypto    Crypto
 	ProvStore provisioningStore
+	factory   orgAlertmanagerFactory
```

Comment:
I don't understand this move, it seemed sensible to me to keep package private fields separate from package public fields?

Reply:
Good catch - I wanted to group it with the other set of interfaces but I fail to notice these are public ones. I'll group it with the private ones.